cluster-up/ssh.sh

```diff
@@ -39,7 +39,7 @@ if [[ $provider_prefix =~ okd.* ]]; then
         echo "no ssh port found for $node"
         exit 1
     fi
-    ssh -lcore -p $port core@127.0.0.1 -i ${ssh_key}
+    ssh -lcore -p $port core@127.0.0.1 -i ${ssh_key} $2
```

Comment:
would it make more sense to shift off the $1 node argument ad pass ```$@``` ?

Reply:
Yeah that works, I am not great with bash arguments, but a shift and $@ works. Let me update the PR.